website/src/db/index.js

```diff
@@ -0,0 +1,115 @@
+import Dexie from 'dexie';
+
+var db = new Dexie("AleoParameters");
+db.version(1).stores({
+    parameters: "name"
+});
+
+const FILES = [
+  // {
+  //   name: 'MintProver',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/mint.prover.11fa6f2'
+  // },
+  // {
+  //   name: 'MintVerifier',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/mint.verifier.e1ae0c8'
+  // },
+  {
+    name: 'TransferProver',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/transfer.prover.837ad21'
+  },
+  {
+    name: 'TransferVerifier',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/transfer.verifier.db46e4c'
+  },
+  // {
+  //   name: 'JoinProver',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/join.prover.369fac2'
+  // },
+  // {
+  //   name: 'JoinVerifier',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/join.verifier.728e514'
+  // },
+  // {
+  //   name: 'SplitProver',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/split.prover.8713930'
+  // },
+  // {
+  //   name: 'SplitVerifier',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/split.verifier.b6bb949'
+  // },
+  {
+    name: 'FeeProver',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/fee.prover.0a31a56'
+  },
+  {
+    name: 'FeeVerifier',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/fee.verifier.2186739'
+  },
+  {
+    name: 'InclusionProver',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/inclusion.prover.b9921c5'
+  },
+  {
+    name: 'InclusionVerifier',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/inclusion.verifier.3f4d6b7'
+  }
+];
+
+export async function getAllSavedFiles() {
+  return await db.table("parameters").toArray();
+}
+
+async function downloadFile(url) {
```

Comment:
IndexedDB is per site so they will need to be downloaded once per site. I don't actually expect them to be used that way as the wallet (and all chrome extensions) have their own IndexedDB instance so you also only have to download these once per extension.

Reply:
Another thought, if a site wants to use these files and doesn't want to download them, we can update the [aleo-wallet-adapter](https://github.com/demox-labs/aleo-wallet-adapter) to serve these files so as long as they have a wallet installed, a website can get the files from there instead of adding another download. 

I don't think it makes sense to add a dependency on the aleo-wallet-adapter here though.

Reply:
I'm not as familiar with chrome extensions, but if you say the extension can store these proving keys in the chrome's extension that's probably good too.

That's an interesting idea to have the wallet serve them, agreed it doesn't make sense to do that here.